package.json

```diff
@@ -73,16 +73,14 @@
     "vitest": "^3.0.5"
   },
   "peerDependencies": {
+    "@typescript-eslint/utils": ">= 8.24.0",
     "eslint": ">= 8.57.0",
     "typescript": ">= 5.0.0",
     "vitest": "*"
   },
   "peerDependenciesMeta": {
     "typescript": {
       "optional": true
-    },
-    "vitest": {
-      "optional": true
     }
```

Comment:
This commit made `vitest` dependency required causing https://github.com/antfu/eslint-config/issues/705
Please restore it